packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -157,7 +161,7 @@ export interface ServerlessClusterProps {
   /**
    * Existing subnet group for the cluster.
    *
-   * @default - a new subnet group will be created.
+   * @default - a new subnet group will be created if a vpc is provided.
```

Comment:
Same comment as for `securityGroups` (what happens if `vpc` is not provided?).

Reply:
👍🏻